src/builtin/tables.scrbl

```diff
@@ -47,53 +67,150 @@ my-table = table: name :: String, age :: Number, favorite-color :: String
 end
   }
 
-Note that @pyret{my-table} is referred to in many of the following examples.
+Entering the name of a table in the interactions window after running the program
+or using the @pyret{print} function will display a formatted version of the table.
+
+@(image "src/builtin/table-print.png")
+  
+@margin-note{Note that @pyret{my-table} is referred to in many of the
+following examples.}
 
   @section[#:tag "s:tables:loading"]{Loading Tables}
 
-Table loading expressions allow for the importing of tables from the
-outside world into Pyret. Currently, only Google Sheets is supported.
-@;(see the @tt{gdrive-sheets} module documentation for details)
-In addition to data sources, the notion of @italic{sanitizers} is used. These
-are used to properly load each entry of the table as the correct type;
-for example, the @pyret{string-sanitizer} in the @tt{data-source} module
-causes each item in its column to be loaded as a string (if it is not a
-string, it is first converted to one). This is illustrated by the following
-example:
+Almost any practical use of a @pyret{Table} will involve loading
```

Comment:
I'll take that parenthetical comment out.  Mainly I was thinking of trying to re-assure the non-CS teacher trying to figure out if Pyret tables are practical in their data analysis context.